packages/wrangler/src/__tests__/deployments.test.ts

```diff
@@ -82,173 +79,151 @@ describe("deployments", () => {
 			it("should log deployments", async () => {
 				writeWranglerToml();
 
-				await runWrangler("deployments list");
+				await runWrangler("deployments list --no-x-versions");
 				expect(std.out).toMatchInlineSnapshot(`
-			"🚧\`wrangler deployments\` is a beta command. Please report any issues to https://github.com/cloudflare/workers-sdk/issues/new/choose
-
-
-			Deployment ID: Constitution-Class-tag:test-name
```

Comment:
I feel like we could keep this line when `--no-x-versions` is set? It's probably not a big deal in this instance since we've logged a warning about the future removal of this line in wrangler for months. But one of the ideas of flipping feature flag defaults is to have an escape-hatch for users to go back to the old behaviour. This doesn't preserve the old behaviour.